src/debezium-service/templates/deployment.yaml

```diff
@@ -0,0 +1,167 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.Values.name}}
+  labels:
```

Comment:
change this to 
`  labels:
        {{- include "pipeline-service.labels" . | nindent 4 }}`